client/src/Components/Card/Card.jsx

```diff
@@ -15,7 +17,7 @@ const MemeCard = ({ value, img, selected, selectCard, id }) => {
 
   return (
     <Card className={selected ? classes.selected : classes.main} variant="outlined">
-      <CardActionArea onClick={onClickHandler}>
+      <CardActionArea onClick={onClickHandler} className={user.isObserver ? classes.cards : null}>
```

Comment:
```suggestion
      <CardActionArea onClick={onClickHandler} className={user.isObserver ? classes.cards : ''}>
```
React jako propsa do className spodziewa się stringa a nie nulla